frontend/src/app/general/auth/basic_auth_token/basic_auth_token.component.html

```diff
@@ -0,0 +1,46 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div style="margin-bottom: 10px; align-items: center">
+  <h2>Token Overview</h2>
+</div>
+<div class="input-container" style="align-items: center; margin-bottom: 10px">
+  <input
+    type="text"
+    [(ngModel)]="tokenDescription"
+    placeholder="Token Description"
+    style="margin-right: 10px"
+  />
+  <button
+    class="self-end mt-auto text-white"
+    (click)="createNewToken(tokenDescription)"
+    [disabled]="!tokenDescription"
+    color="primary"
+    mat-raised-button
+  >
+    Create Token
+  </button>
+  <div *ngIf="password" style="margin-bottom: 10px">
+    <div>Generated Password: {{ password }}</div>
+    <button
+      class="font-mono cursor-copy"
+      [cdkCopyToClipboard]="password"
+      (click)="showClipboardMessage()"
+    >
+      <mat-icon>content_copy</mat-icon>
+    </button>
+  </div>
```

Comment:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/6a788c17-024c-4339-9650-8ff3f3e7d203)

The Gitlab UI also looks good. Especially the comment "Make sure you save it - you won't be able to access it again." is important.

Reply:
now it looks like
![Bildschirmfoto 2023-09-22 um 11 10 08](https://github.com/DSD-DBS/capella-collab-manager/assets/49535291/3bb2f9f1-8343-4742-a86b-b7e1d4f4de01)

was that what you imagined it to look like? I am a bit wondering about the mat-card because now it is just another card as all the tokens shown as well but maybe it is also fine

Reply:
changed it to
![Bildschirmfoto 2023-09-25 um 09 03 45](https://github.com/DSD-DBS/capella-collab-manager/assets/49535291/aaa85857-c72c-43b6-a0c3-aeb61069563f)